lapack.go

```diff
@@ -11,22 +11,57 @@ const None = 'N'
 type Job byte
 
 const (
-	All       (Job) = 'A'
-	Slim      (Job) = 'S'
-	Overwrite (Job) = 'O'
+	All       Job = 'A'
+	Slim      Job = 'S'
+	Overwrite Job = 'O'
 )
 
 type CompSV byte
 
 const (
-	Compact  (CompSV) = 'P'
-	Explicit (CompSV) = 'I'
+	Compact  CompSV = 'P'
+	Explicit CompSV = 'I'
 )
 
+// Complex128 defines the complex128 interface for the Lapack function. This interface
+// contains the complex128 functions supported by gonum.
+type Complex128 interface{}
+
 // Float64 defines the float64 interface for the Lapack function. This interface
-// contains the functions needed in the gonum suite.
+// contains the float64 functions supported by gonum.
 type Float64 interface {
 	Dpotrf(ul blas.Uplo, n int, a []float64, lda int) (ok bool)
 }
 
-type Complex128 interface{}
+// Direct specifies the direction of the multiplication for the Householder matrix.
+type Direct byte
+
+const (
+	Forward  Direct = 'F' // Reflectors are right-multiplied, H_1 * H_2 * ... * H_k
+	Backward Direct = 'B' // Reflectors are left-multiplied, H_k * ... * H_2 * H_1
+)
+
+// StoreV indicates the storage direction of elementary reflectors.
+type StoreV byte
+
+const (
+	ColumnWise StoreV = 'C' // Reflector stored in a column of the matrix.
+	RowWise    StoreV = 'R' // Reflector stored in a row of the matrix.
+)
+
+// MatrixNorm represents the kind of matrix norm to compute.
+type MatrixNorm byte
+
+const (
+	MaxAbs       MatrixNorm = 'M' // max(abs(A(i,j)))  ('M')
+	MaxColumnSum MatrixNorm = 'O' // Maximum column sum (one norm) ('1', 'O')
```

Comment:
Yes. I found it useful when looking at fortran Lapack code. The actual lapack code has things like "if norm == '1'", and it's nice to have a reference what that means. I thought it would be useful to others with fortran experience, but I can delete if you'd prefer.